Formula/a/argtable3.rb

```diff
@@ -0,0 +1,42 @@
+class Argtable3 < Formula
+  desc "ANSI C library for parsing GNU-style command-line options"
+  homepage "https://www.argtable.org"
+  url "https://github.com/argtable/argtable3/archive/refs/tags/v3.2.2.f25c624.tar.gz"
```

Comment:
Version number scanning [incorrectly returns `624`](https://github.com/Homebrew/homebrew-core/actions/runs/11196858337/job/31126187386?pr=192923#step:3:43) so we need to override it:

```suggestion
  url "https://github.com/argtable/argtable3/archive/refs/tags/v3.2.2.f25c624.tar.gz"
  version "3.2.2"
```